sys/include/pm_layered.h

```diff
@@ -35,11 +35,14 @@
 #define PM_LAYERED_H
 
 #include "assert.h"
-#include "periph/pm.h"
 #include "periph_cpu.h"
 
 #ifdef __cplusplus
- extern "C" {
+extern "C" {
+#endif
+
+#ifndef PROVIDES_PM_OFF
```

Comment:
I personally don't like mixing PROVIDED_PM_FF and PROVIDES_PM_LAYERED_OFF. Can we get rid of this? Or is this required?

Reply:
IMHO this is required, as each of those defines overrides a specific function in the 'fallback hierarchy'. As I see it, this is the price to pay for having multiple tiers of fallback functions.

But maybe we come up with an even better idea?